### PR TITLE
Add Window.scrollTo Task

### PR DESCRIPTION
**Pull request by stevenyap**
This PR is to add a Window.scrollTo task so that user can make the window scroll.

The use case I am trying to solve is to scroll the window to the top whenever the user navigates in a SPA. The problem is that if the user is at the bottom of the page, navigating to another page will not scroll the user to the top but leaves the user at the bottom of the new page.

My vim with elm-format causes a lot of changes in the formatting which I have separated out into another commit.

### CODE REVIEW

File: src/Native/Window.js [hunk 1]
```diff
@@ -8,8 +8,16 @@ var size = _elm_lang$core$Native_Scheduler.nativeBinding(function(callback)	{
...
 
+var scrollTo = function(x, y) {
+  return _elm_lang$core$Native_Scheduler.nativeBinding(function(callback)	{
+    window.scrollTo(x, y);
+    callback(_elm_lang$core$Native_Scheduler.succeed());
+  });
+};
+  
 return {
+  scrollTo: F2(scrollTo),
```
Comment (TARPTAEYA) [inline src/Native/Window.js]: this is not indented properly

Comment (tarptaeya) [inline src/Native/Window.js]: This is not indented properly, also since scrollTo comes after size, IMHO it must be after that

**Review by tarptaeya — COMMENTED** [review]
IMHO the changed formatting is not homogneous with other project's files, Do you really want to change them?

**Comment by stevenyap** [comment]
Hi I have updated the indentation for the JS.

As for the changed formatting, I assume you are referring to `Window.elm`, this is auto-formatted based on elm-format in my Vim editor. Since Window.elm last commit is 2 years ago, I suppose that the formatting is outdated and hence we should update the formatting.

**Comment by evancz** [comment]
This is available with a the `setViewport` function in elm/browser with Elm 0.19

**Pull request closed without merging**